### PR TITLE
Introduce CoreText control

### PR DESCRIPTION
**Pull request by johnny9**
A Text item with the most commonly used property values.

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/<PR>)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/<PR>)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/<PR>)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/<PR>)

### CODE REVIEW

File: src/qml/controls/ExternalLink.qml [hunk 1]
```diff
@@ -45,9 +45,7 @@ AbstractButton {
             Layout.fillWidth: true
             active: root.description.length > 0
             visible: active
-            sourceComponent: Text {
-                font.family: "Inter"
-                font.styleName: "Regular"
+            sourceComponent: CoreText {
```
Comment (jarolrod) [inline src/qml/controls/ExternalLink.qml:48]:
```diff

diff --git a/src/qml/controls/CoreText.qml b/src/qml/controls/CoreText.qml
index 1029cd0af..56e122a9a 100644
--- a/src/qml/controls/CoreText.qml
+++ b/src/qml/controls/CoreText.qml
@@ -7,11 +7,12 @@ import QtQuick.Controls 2.15
 
 Text {
     property bool bold: false
+    property bool wrap: true
     color: Theme.color.white
     font.family: "Inter"
     font.styleName: bold ? "Semi Bold" : "Regular"
     font.pixelSize: 13
     horizontalAlignment: Text.AlignHCenter
     verticalAlignment: Text.AlignVCenter
-    wrapMode: Text.WordWrap
+    wrapMode: root.wrap ? Text.WordWrap : Text.NoWrap
 }
diff --git a/src/qml/controls/ExternalLink.qml b/src/qml/controls/ExternalLink.qml
index ab10d1b89..2e189037a 100644
--- a/src/qml/controls/ExternalLink.qml
+++ b/src/qml/controls/ExternalLink.qml
@@ -48,6 +48,7 @@ AbstractButton {
             sourceComponent: CoreText {
                 font.pixelSize: root.descriptionSize
                 color: root.textColor
+                wrap: false
                 textFormat: Text.RichText
                 text: root.description
```

Comment (jarolrod) [reply src/qml/controls/ExternalLink.qml:48]: Something like the above would fix the [wrapping issue](https://github.com/bitcoin-core/gui-qml/pull/268#pullrequestreview-1308709522) I noticed on macOS

**Comment by johnny9** [comment]
Update from[ 13558f1 ](https://github.com/bitcoin-core/gui-qml/commit/13558f14ff35c5398c15b43baf82a32fe707c079)to 32a11f3: 
 - Added wrap property as suggested by jarolrod

**Review by jarolrod — APPROVED** [review]
ACK 32a11f3f6ce1867383cbbbbcce78b07121e0bd24

**Review by jarolrod — COMMENTED** [review]
screenshots, master on left, pr on right

| continue button | external link | nav button | network indicator | outline button | text button |
| --------------- | ------------- | ---------- | ----------------- | -------------- | ----------- |
| continue-button | external-link | nav-button | network-indicator | outline-button | text-button |

**Pull request merged**